binaries/workerserver/main.go

```diff
@@ -8,7 +8,8 @@ import (
 
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/scootdev/scoot/common/endpoints"
-	"github.com/scootdev/scoot/runner/execer/fake"
+	"github.com/scootdev/scoot/runner/execer/execers"
+	osexec "github.com/scootdev/scoot/runner/execer/os"
```

Comment:
BTW, I'd suggest the positive example of https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Executors.html , where Foos is used to store many Foos, but not all.